server/src/testFixtures/java/io/crate/testing/LogicalPlanAssert.java

```diff
@@ -24,22 +24,36 @@
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 import org.assertj.core.api.AbstractAssert;
+import org.jetbrains.annotations.Nullable;
 
 import io.crate.planner.operators.LogicalPlan;
 import io.crate.planner.operators.PrintContext;
+import io.crate.planner.optimizer.costs.PlanStats;
 
 public class LogicalPlanAssert extends AbstractAssert<LogicalPlanAssert, LogicalPlan> {
 
+    @Nullable
+    private final PlanStats planStats;
```

Comment:
This makes sure estimates can be printed in the tests if the plan stats are provided.